components/script/dom/document.rs

```diff
@@ -516,6 +516,25 @@ impl<'a> DocumentMethods for JSRef<'a, Document> {
     fn URL(self) -> DOMString {
         self.url().serialize()
     }
+    
+    // https://html.spec.whatwg.org/multipage/interaction.html#dom-document-activeelement
+    fn GetActiveElement(self) -> Option<Temporary<Element>> {
+        // TODO: Step 2.
+
+        match self.get_focused_element() {
+            Some(element) => Some(element),     // Step 3. and 4.
+            None => match self.GetBody() {      // Step 5.
+                Some(body) => Some(ElementCast::from_temporary(body)),
+                None => {
+                    let node = NodeCast::from_ref(self);
+                    match node.GetFirstChild() {
+                        Some(_) => ElementCast::to_ref(node).and_then(ElementHelpers::get_root_element),
```

Comment:
Sorry, I'm a bit confused now. Here is part of our conversation on irc:
> 11:51	mskrzypkows	Hi, am I right that Document::GetDocumentElement returns root element as described in https://html.spec.whatwg.org/multipage/infrastructure.html#root-element ?
> 11:53	Ms2ger	No
> 11:53	Ms2ger	It's https://html.spec.whatwg.org/multipage/infrastructure.html#root-element-of-a-document-object

I thought that because of the link in spec: https://html.spec.whatwg.org/multipage/infrastructure.html#root-element and I have to implement ElementHelpers::get_root_element as I did.
To sum up: now I understand that in spec https://html.spec.whatwg.org/multipage/infrastructure.html#root-element in section https://html.spec.whatwg.org/multipage/interaction.html#dom-document-activeelement means the Document object's root element and I have to use `Document::GetDocumentElement` to get it, right?
Should I delete ElementHelpers::get_root_element method?